resources/home/dnanexus/utils/util_functions.py

```
@@ -1,22 +1,124 @@
+        project_name (str): name of DNAnexus project
+        project_id (str): ID of DNAnexus project
+        job_output (str): destination folder set for the job
+    """
+    project_id = os.environ.get("DX_PROJECT_CONTEXT_ID")
+
+    # Set the environment context to allow upload
+    dxpy.set_workspace_id(project_id)
+
+    # Get output folder set for this job
+    job_output = dxpy.bindings.dxjob.DXJob(
+        os.environ.get("DX_JOB_ID")
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)